src/SymfonyRequirements.php

```diff
@@ -360,6 +360,14 @@ function_exists('posix_isatty'),
 
         $this->addPhpConfigRecommendation('session.auto_start', false);
 
+        $this->addPhpConfigRecommendation('post_max_size', function () {
+            return ini_get('post_max_size') < ini_get('memory_limit');
+        }, true, '"memory_limit" should be superior than "post_max_size".');
```

Comment:
"should be larger than" is probably better

Also the values of the ini's are not always integers. See `getRealpathCacheSize`